layouts/guide/single.html

```diff
@@ -57,6 +57,9 @@ <h4>AC Transit Ward: {{ .Key }}</h4>
             {{ end }}
             {{ range where (.Data.Pages.GroupByParam "Office") ".Key" "==" "BART Board of Directors" }}
             <div>
+              {{ $districts := where (.Pages.GroupByParam "district") ".Key" "in" $.Page.Params.bart_districts }}
+              {{ $count := len $districts }}
+              {{ if gt $count 0 }}
```

Comment:
This shouldn't be necessary because there's a BART district for everyone...  But right now that is not apparently the case, and we should fix that in another ticket.

Reply:
Is `$.Page.Params.bart_districts` not populated?

Reply:
Wait no it is and it's fine, just no candidates in BART D4 like I was looking at.  :D